libs/user-facing-errors/src/schema_engine.rs

```diff
@@ -9,13 +9,14 @@ pub struct DatabaseCreationFailed {
 }
 
 /// [spec](https://github.com/prisma/specs/tree/master/errors#p3001-destructive-migration-detected)
+/// No longer used.
```

Comment:
Added the comment since one of the unused errors below also has one.

What's our plan for these? Do we want to remove them at some point or is there reason to keep them?

Reply:
I'm all for removing, if so we need to also remove it from docs (or mark it unused there is better I guess)
https://github.com/prisma/docs/blob/d28bf801258e34b4a699014b3cb7c43362ecf1d2/content/400-reference/200-api-reference/250-error-reference.mdx?plain=1#L326

Reply:
Doesn't have to be done now though, it's not that important

Reply:
Is it intentionally not used any more? Or was it maybe accidentally removed only?